project/src/main/kotlin/org/ionproject/core/common/Uri.kt

```diff
@@ -0,0 +1,21 @@
+package org.ionproject.core.common
+
+import org.springframework.web.util.UriTemplate
+
+object Uri {
+    private const val version = "/v0"
+
+    const val courses = "${version}/courses"
+    const val courseByAcr = "$courses/{course_id}"
+    const val klasses = "$courseByAcr/classes"
+    const val klassByTerm = "$klasses/{class_id}"
+    const val classSectionById = "$klassByTerm/{classSection_id}"
+    const val calendarByClass = "$klassByTerm/calendar"
+    const val calendarByClassSection = "$classSectionById/calendar"
+
+    val calendarByClassTemplate = UriTemplate(calendarByClass)
+    val calendarByClassSectionTemplate = UriTemplate(calendarByClassSection)
+
+    fun forCalendarByClass(courseId: Int, classId: Int) = calendarByClassTemplate.expand(courseId, classId)
```

Comment:
I thought we had changed to using just IDs right?

Reply:
We changed the Course resource's identifier to a decimal because of internationalization. Since a single course may have multiple acronyms and names for different languages, the path will remain the exact same for all of them (e.g. instead of /courses/wad or /courses/daw, we might use /couses/1). Since a class is identified by its calendar term this problem does not exist.